docs/manual/other-topics/upgrade-to-v7.md

```diff
@@ -16,6 +25,92 @@ As a result, the manual typings that were formerly best-effort guesses on top of
 have been removed and all typings are now directly retrieved from the actual TypeScript code.
 You'll likely find many tiny differences which however should be easy to fix.
 
+#### Changes to class-based Model typings
+
+Typings for Models have been greatly simplified. Instead of having to write something like the following:
+
+```typescript
+import { Model, Optional } from 'sequelize';
+
+type UserAttributes = {
+  id: number;
+  firstName: string;
+};
+
+type UserCreationAttributes = Optional<UserAttributes, 'id'>;
+
+class User extends Model<UserAttributes, UserCreationAttributes> {
+  declare id: number;
+  declare firstName: string;
+}
+```
+
+You instead need to write it like this:
+
+```typescript
+import { Model, CreationOptional } from 'sequelize';
+
+class User extends Model<User> {
```

Comment:
Since I have never used this syntax before, I have no clue but I think one would have to call the Model's init function afterwards somehow?

Can you add that to the example?